tests/sentry/api/endpoints/test_group_ai_summary.py

```diff
@@ -86,3 +108,8 @@ def test_ai_summary_call_seer(self, mock_get_event, mock_post):
             mock_response.json.return_value, snake_to_camel_case
         )
         mock_post.assert_called_once()
+        mock_cache.set.assert_called_once_with(
+            f"ai-group-summary:{group.id}",
+            mock_response.json.return_value,
+            timeout=60 * 60 * 24 * 7,  # 7 days
+        )
```

Comment:
wanna add one test where the cache isn't mocked?

Reply:
done